### ISSUE TITLE
Unclear error message when building a project that contains dashes

### ISSUE THREAD

**Issue by thebf** [zombodb/pgx#67]
[error] library file not found in: /home/user/pgx-s3sign/target/release
 
Could suggest a correction.

**Comment by eeeebbbbrrrr** [comment]
looks like the root cause for this is already in-place:

```
$ cargo pgx new some-thing
     [error] Extension name must be in the set of [a-z0-9_]
```

but I'll add a check for this to error things out down the pipeline too

**Comment by eeeebbbbrrrr** [comment]
<img width="1342" alt="Screen Shot 2020-12-18 at 3 59 00 PM" src="https://user-images.githubusercontent.com/5117238/102668758-fb54f880-4149-11eb-8613-a8568ddc1d31.png">

I think this takes care of it if you manage to get yourself into this situation in some sneaky, "I'm smarter than @eeeebbbbrrrr" kind of way.

**Comment by eeeebbbbrrrr** [comment]
released in 0.1.7
